cli_tests/test_list_debuggees_command.py

```diff
@@ -0,0 +1,266 @@
+    'labels': {
+        'version': 'v1'
+    },
+    'description': 'desc 1'
+}, ['default - v1', '123', 'desc 1'])
+
+debuggee_missing_version = ({
+    'id': '123',
+    'labels': {
+        'module': 'app123',
+    },
+    'description': 'desc 1'
+}, ['app123 - ', '123', 'desc 1'])
+
+debuggee_missing_description = ({
+    'id': '123',
+    'labels': {
+        'module': 'app123',
+        'version': 'v1'
+    },
+}, ['app123 - v1', '123', ''])
+
+debuggee1 = ({
+    'id': '123',
+    'labels': {
+        'module': 'app123',
+        'version': 'v1'
+    },
+    'description': 'desc 1'
+}, ['app123 - v1', '123', 'desc 1'])
+
+debuggee2 = ({
+    'id': '456',
+    'labels': {
+        'module': 'app456',
+        'version': 'v2'
+    },
+    'description': 'desc 2'
+}, ['app456 - v2', '456', 'desc 2'])
+
+debuggee3 = ({
+    'agentVersion': 'google.com/node-gcp/v6.0.0',
+    'description': 'node index.js module:test-app version:v1',
+    'id': '97b0090602ceb5a506f1cf7dff02524f',
+    'labels': {
+        'V8_version': '7.8.279.23-node.56',
+        'agent_name': '@google-cloud/debug-agent',
+        'agent_version': '6.0.0',
+        'main_script': 'index.js',
+        'module': 'test-app',
+        'node_version': '12.22.5',
+        'platform': 'default',
+        'process_title': 'node',
+        'projectid': 'unknown',
+        'version': 'v1'
+    },
+    'project': 'unknown',
+    'uniquifier': 'ecbb07aaba5ad405ad7667bf3eacbd874e23a2f5'
+}, [
+    'test-app - v1', '97b0090602ceb5a506f1cf7dff02524f',
+    'node index.js module:test-app version:v1'
+])
+
+
+def debuggees_to_dict(debuggees):
+  return dict((d['id'], d) for d in debuggees)
+
+
+class ListDebuggeesCommandTests(unittest.TestCase):
+  """ Contains the unit tests for the ListDebuggeesCommand class.
+  """
+
+  def setUp(self):
+    common_parsers = cli_common_arguments.CommonArgumentParsers()
+    required_parsers = cli_common_arguments.RequiredArgumentParsers().parsers
+
+    self.args_parser = argparse.ArgumentParser()
+    self.args_subparsers = self.args_parser.add_subparsers()
+    self.list_debuggees = list_debuggees_command.ListDebuggeesCommand()
+    self.list_debuggees.register(self.args_subparsers, required_parsers,
+                                 common_parsers)
+    self.cli_services = MagicMock()
+
+    self.user_output_mock = MagicMock()
+    self.cli_services.user_output = self.user_output_mock
+
+    self.rtdb_service_mock = MagicMock()
+    self.cli_services.get_snapshot_debugger_rtdb_service = MagicMock(
+        return_value=self.rtdb_service_mock)
+
+  def test_get_debuggees_called_once(self):
+    args = self.args_parser.parse_args(['list_debuggees'])
+    self.rtdb_service_mock.get_debuggees = MagicMock(return_value={})
+
+    self.list_debuggees.cmd(args, self.cli_services)
+    self.rtdb_service_mock.get_debuggees.assert_called_once()
+
+  def test_output_format_default(self):
+    expected_headers = ['Name', 'ID', 'Description']
+
+    # yapf: disable
```

Comment:
Is there a way to explain why the formatting is disabled?

Reply:
I tweaked it, and added a comment. Let me know if that's what you had in mind.